react/features/face-landmarks/functions.js

```diff
@@ -60,44 +60,44 @@ export function sendFaceBoxToParticipants(
 }
 
 /**
- * Sends the facial expression with its duration to xmpp server.
+ * Sends the face expression with its duration to xmpp server.
  *
  * @param {Object} conference - The current conference.
- * @param  {string} facialExpression - Facial expression to be sent.
- * @param {number} duration - The duration of the facial expression in seconds.
+ * @param  {string} faceExpression - Face expression to be sent.
+ * @param {number} duration - The duration of the face expression in seconds.
  * @returns {void}
  */
-export function sendFacialExpressionToServer(
+export function sendFaceExpressionToServer(
         conference: Object,
-        facialExpression: string,
+        faceExpression: string,
         duration: number
 ): void {
     try {
-        conference.sendFacialExpression({
-            facialExpression,
+        conference.sendFaceExpression({
```

Comment:
Didn't you rename this to sendFaceLandmark?

Reply:
no

Reply:
actually yes..